plugins/outputs/zabbix/utils_test.go

```diff
@@ -0,0 +1,184 @@
+package zabbix
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestLLDKey(t *testing.T) {
```

Comment:
Same for this file, you might want to move the tests to the test-files containing the implementations...

Reply:
Done in d48a74f96.
Also deleted `generateLLDKey`, no longer used.